content/docs/deployment/csminstallationwizard/_index.md

```diff
@@ -83,7 +83,7 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 7. If your Kubernetes distribution doesn't have Volume Snapshot feature enabled, Refer to [this section](../../snapshots) to install the Volume Snapshot CRDs and the default snapshot controller.
 
-8. If Replication is checked in the wizard, only the sidecar is enabled. Refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters and the replication controller.
+8. If Replication is checked in the wizard, refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters.
```

Comment:
'only sidecar is enabled' seems to be applicable only for Authorization module. Please see earlier version of doc - https://dell.github.io/csm-docs/v1/deployment/csminstallationwizard/
If this is the case, could you revert the change for other modules?